warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -393,7 +393,7 @@ private DateIndexHelper getDateIndexHelper(final ShardQueryConfiguration config)
         }
         
         final QueryStopwatch timers = config.getTimers();
-        
+        System.out.println(PrintingVisitor.formattedQueryString(queryTree));
```

Comment:
Should this be removed or written to the log?

Reply:
will remove